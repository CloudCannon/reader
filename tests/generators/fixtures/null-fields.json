{
	"source": null,
	"data_config": null,
	"collections_config": null,
	"_comments": null,
	"_options": null,
	"_structures": null,
	"_select_data": null,
	"generator": null,
	"source_editor": null,
	"paths": {
		"uploads": null
	},
	"base_url": null
}
