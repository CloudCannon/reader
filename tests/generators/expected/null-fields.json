{
	"source": "",
	"data_config": {},
	"collections_config": {},
	"_comments": {},
	"_options": {},
	"_structures": {},
	"_select_data": {},
	"generator": {},
	"source_editor": {},
	"paths": {
	  "uploads": null
	},
	"base_url": "",
	"time": "2000-11-22T00:00:00.000Z",
	"cloudcannon": {
	  "name": "cloudcannon-reader",
	  "version": "0.0.1"
	},
	"version": "0.0.3",
	"data": {},
	"collections": {}
  }
